packages/samples/hello/java/build.gradle

```diff
@@ -0,0 +1,13 @@
+apply plugin: 'java'
```

Comment:
@houshengbo Because the original sample is using Gradle to build. So it follows the graddle structure. Refer to https://github.com/openwhisk/openwhisk/tree/master/catalog/samples/helloJava